sycl/CMakeLists.txt

```diff
@@ -343,6 +343,9 @@ if(SYCL_BUILD_PI_ESIMD_EMULATOR)
   else()
     list(APPEND SYCL_TOOLCHAIN_DEPLOY_COMPONENTS libcmrt-sos)
   endif()
+else()
+  ## TODO : Remove depedency on cm_rt.h for non-esimd-emulator execution
+  file (TOUCH ${SYCL_INCLUDE_BUILD_DIR}/sycl/CL/cm_rt.h)
```

Comment:
Why do we need empty cm_rt.h?

Reply:
Without this dummy `cm_rt.h` from toolchain built with no esimd_emulator support, kernel compilation targeting host device fails as the file is included from `memory_intrin.h` under `#ifndef __SYCL_DEVICE_ONLY__`.